tests/unit/script/htmlimageelement.rs

```diff
@@ -0,0 +1,16 @@
+
+use script::test::sizes::parse_a_sizes_attribute;
+use script::test::DOMString;
+
+#[test]
+fn empty_vector() {
+    assert!(parse_a_sizes_attribute(DOMString::new(), None).is_ok());
```

Comment:
@Rakhisharma  As discussed, the test case depicts the situation where sizes is an empty string (`DOMString::new()`), so here we should also check if the vector contains a single `Size` with the value `100vw`. (i.e., the default value returned from spec algorithm)